cibyl/sources/elasticsearch/api.py

```diff
@@ -459,6 +466,15 @@ def get_tests(self, **kwargs):
                           )
                 continue
 
+            if test_duration_arguments and \
+               not self.match_filter_test_by_duration(
+                   test_duration,
+                   test_duration_arguments):
+                continue
+
+            if test_duration:
```

Comment:
I would add a comment here mentioning that it converts into ms, instead of having just a naked number

Reply:
Done. Thanks!